api/v1/configurationpolicy_types.go

```diff
@@ -51,6 +51,9 @@ const (
 
 	// UnknownCompliancy is an ComplianceState
 	UnknownCompliancy ComplianceState = "UnknownCompliancy"
+
+	// Terminating is an ComplianceState
+	Terminating ComplianceState = "Terminating"
```

Comment:
Since we want this in the `kubectl` output, can we also add compliance state as a printercolumn? Like https://github.com/open-cluster-management-io/governance-policy-propagator/blob/main/api/v1/policy_types.go#L102, for reference